model_analyzer/triton/server/server_docker.py

```diff
@@ -68,6 +68,9 @@ def __init__(self, image, config, gpus, log_path, mounts, labels, shm_size):
         assert self._server_config['model-repository'], \
             "Triton Server requires --model-repository argument to be set."
 
+        logger.info(f"Pulling docker image {self._tritonserver_image}")
```

Comment:
(nit) Do we want this to print out if the pull is skipped?
I think it has the potential to be confusing for the user, until they look at the time stamps.
This is boy scout change at most. I am not going to gate this merge for this change.

Reply:
no, I want it to be clear that docker image is being used (because they may realize that they don't want the default)